docs/page.css

```diff
@@ -152,7 +152,7 @@ ul code {
 	margin: 16px 0;
 }
 
-code.inline {
+code.inline, code[inline] {
```

Comment:
Small update for convenience, allows writing `<code inline>...</code>` instead of `<code class="inline">...</code>`.